cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1153,10 +1153,10 @@ private void updateCluster(VertxTestContext context, Kafka originalAssembly, Kaf
 
         if (originalCruiseControl != null) {
             when(mockDepOps.get(clusterNamespace, CruiseControlResources.componentName(clusterName))).thenReturn(
-                    originalCruiseControl.generateDeployment(true, null, null)
+                    originalCruiseControl.generateDeployment(emptyMap(), true, null, null)
```

Comment:
Can't you use `Map.of()`? We normally tend to use that rather than `emptyMap()` in newer code. Same as below.

Reply:
Done